main.js

```diff
@@ -101,6 +102,24 @@ async function init() {
           message.reply('Could not log hours :( yell at ernest');
         }
         break;
+
+      case '!hiscores':
+      case '!highscores':
+      case '!leaderboards':
+        const totalHours = await getTotalHours(googleClient);
+        const maxHiscoresLength = 10;
```

Comment:
Let's use uppercase for these kinds of constant parameters.

```suggestion
        const MAX_HISCORES_LENGTH = 10;
```

Reply:
>  I don't know where it should be put

Yup, putting it here will do for now.

The three places for constants are:
- `config.json` are settings associated with the environment you're running on. E.g., my development environment will be using a different discord bot token compared with your development environment, both of which will be different to the production environment which will be using the actual discord bot's token.
- Google Sheets settings page: things that are likely to change, such as the milestone values. Putting them in the spreadsheet makes it easy for non-tech-savvy users to change.
- Inline with the code: things that probably won't change between environments and probably won't change over time. We can always move constants from here to the Google Sheets if we feel like it.

Reply:
Yeah, forgot to change it from lowercase from when it was a function argument